custom_components/octopus_intelligent/graphql_client.py

```diff
@@ -191,7 +191,7 @@ async def __async_cancel_boost_charge(self, session, account_id: str):
     result = await session.execute(query, variable_values=params, operation_name="deleteBoostCharge")
     return result['deleteBoostCharge']
 
-  async def __async_get_accounts(session):
+  async def __async_get_accounts(self, session):
```

Comment:
The `start()` function was not being executed, as it wasn’t awaited. When `start()` is awaited (the prominent fix in this PR), it executes other code and reveals other issues, like this issue (`__async_get_accounts() takes 1 positional argument but 2 were given`):
```
2024-02-25 23:25:02.662 ERROR (MainThread) [custom_components.octopus_intelligent.octopus_intelligent_system] Authentication failed : OctopusEnergyGraphQLClient.__async_get_accounts() takes 1 positional argument but 2 were given. You may need to check your token or create a new app in the gardena api and use the new token.
```